src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTask(Task t) {
+        tasks.add(t);
+    }
+
+    public void deleteTask(int index) {
+        tasks.remove(index - 1);
+    }
+
+    public Task getTask(int index) {
+        return tasks.get(index - 1);
+    }
+
+    public int getTasksSize() {
+        return tasks.size();
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return tasks;
+    }
+
+    public TaskList find(String keyword) {
```

Comment:
Maybe could rename to findTask() to keep in convention with the rest of your methods?